src/css/application.css

```diff
@@ -211,8 +210,7 @@ body {
 }
 
 .top-bar__menu-item_icons {
-  letter-spacing: 1.5em;
-  margin-right: -1.5em;
```

Comment:
If we do change the `letter-spacing`, we should still have an offsetting negative margin so that the icons are centered (because `letter-spacing` creates space *after* every letter, not just between them, which is DUMB)